src/CMakeLists.txt

```diff
@@ -1045,3 +1045,6 @@ add_subdirectory(script)
 if(WITH_EMBEDDED)
   add_subdirectory(libcephd)
 endif()
+
+configure_file(${CMAKE_SOURCE_DIR}/src/upstart/ceph-osd.conf.in
+  ${CMAKE_SOURCE_DIR}/src/upstart/ceph-osd.conf @ONLY)
```

Comment:
@johnlinp please do not put the generated files into `${CMAKE_SOURCE_DIR}`, otherwise the out-of-source build is sort of broken. we should put them into `${CMAKE_BINARY_DIR}`. and you might want to update the install script (ceph.spec.in and debian/rules) to reflect this change.